thyrel-web/src/components/home/PlayerForm.tsx

```diff
@@ -33,9 +36,8 @@ export default function PlayerForm({ identifier }: { identifier?: string }) {
       (player: Player) => {
         setLoading(false);
         if (player.token?.tokenKey) {
-          Notification['success']({
-            title: 'Room successfully created.',
-            description: 'Invite your friends.',
+          enqueueSnackbar('Room successfully created 🙌 Invite your friends.', {
```

Comment:
```suggestion
          enqueueSnackbar('Room successfully created 🙌', {
```

notif courte = bonne notif